Driv3rs.py

```diff
@@ -246,14 +239,25 @@ def md5hash(bytes):
     else:
         drivers_list[i]['version'] = V + '.' + v0 + v1
 
-    # calculate distance from beginning of current driver to
-    # next driver. reposition pointer to beginning of current
-    # driver. send number of bytes to read to hashing function
-    ## DISABLED FOR DEBUGGING ## EXPECT BLANK FIELDS IN the CSV
-    #bytes_to_read = drivers_list[i]['comment_start'] + \
-    #drivers_list[+1]['comment_start'] - 4
-    #SOSfile.seek(drivers_list[i]['comment_start'] + 4,0)
-    #drivers_list[i]['md5'] = md5hash(bytes_to_read)
+    # device configuration block length
```

Comment:
According to the official manual, the Configuration Block is optional. Also, I don't see anything that indicates a common byte-length, so I'm not clear on where you're getting 2 bytes from...?